x-pack/filebeat/module/gcp/audit/config/pipeline.js

```diff
@@ -2,6 +2,24 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+function traverse(o,func) {
```

Comment:
I think this is nice, I'm just a bit scared we (for whatever reason) end up in an infinite loop somehow..

Reply:
Quick question: we talked about the actual query being in either `json.serviceData.jobGetQueryResultsRequest.job.jobConfiguration.query.query` or in `json.serviceData.jobGetQueryResultsResponse.job.jobConfiguration.query.query` right? Is there another potential place in the json where the query _might_ be which justifies this extra complexity here?